yelp_package/extra_requirements_yelp.txt

```diff
@@ -1,4 +1,5 @@
 --extra-index-url=https://pypi.yelpcorp.com/simple
+git+git@git.yelpcorp.com:python_packages/yelp_paasta_helpers
```

Comment:
should be git+ssh no?

Can you run a `make itest_trusty` in yelp to test? (travis can't test)